planning/autoware_obstacle_cruise_planner/src/node.cpp

```diff
@@ -520,11 +582,12 @@ void ObstacleCruisePlannerNode::onTrajectory(const Trajectory::ConstSharedPtr ms
   //    (1) with a proper label
   //    (2) in front of ego
   //    (3) not too far from trajectory
-  const auto target_obstacles = convertToObstacles(ego_odom, objects, traj_points);
+  const auto target_obstacles =
+    convertToObstacles(ego_odom, objects_ptr, pointcloud_ptr, traj_points, msg->header);
```

Comment:
Wouldn't it be cleaner to call convertToObstacles()  by type? like
const auto target_obstacles =
convertToObstacles(ego_odom, objects_ptr, traj_points, msg->header) + convertToObstacles(ego_odom, pointcloud_ptr, traj_points, msg->header);

I think my misunderstanding comment may have gotten you to make this change. Sorry.

Reply:
Fixed: https://github.com/autowarefoundation/autoware.universe/pull/6907/commits/5d5ec3f6d0b213d2ee5501b202870a35c2807beb